conf/task_defaults/naipchesapeake.yaml

```diff
@@ -10,11 +10,11 @@ experiment:
     learning_rate_schedule_patience: 2
     in_channels: 4
     num_classes: 13
-    num_filters: 64
+    num_filters: 1
     ignore_zeros: False
   datamodule:
     naip_root_dir: "tests/data/naip"
     chesapeake_root_dir: "tests/data/chesapeake/BAYWIDE"
-    batch_size: 32
+    batch_size: 2
```

Comment:
With a batch size of 1 (what I'm using on all other tests) this breaks and I don't know why.